src/calc_matrix_elements_module.f90

```diff
@@ -207,7 +207,6 @@ subroutine get_matrix_elements_new(myid,rem_bucket,matM,gridone,gridtwo)
           allocate(acc_block(n_dim_two*n_dim_one),STAT=stat)
           if(stat/=0) call cq_abort('Error allocating memory in get_matrix_elements: ',n_dim_one,n_dim_two)
           acc_block = zero
-          call reg_alloc_mem(area_integn,n_dim_two*n_dim_one,type_dbl)
```

Comment:
Is removing this related to the if statments?

Reply:
Not strictly, no - I was maybe a little over-zealous in tidying! I can put this back if we want and remove it in a separate PR...

Reply:
(I have removed a call to a subroutine which registers the amount of memory allocated and the equivalent subroutine which notes that it's been freed, which is really not needed here; it doesn't actually allocate anything)